From gregkh@suse.de Thu May 15 21:04:53 2008
X-Envelope-From: <stable-commits-owner@vger.kernel.org>
X-Envelope-To: <vger@sl-h.de>
X-Delivery-Time: 1210881087
X-UID: 237684
Return-Path: <stable-commits-owner@vger.kernel.org>
X-RZG-CLASS-ID: mi
Received: from vger.kernel.org ([209.132.176.167])
	by mailin.webmailer.de (christine mi55) (RZmta 16.35)
	with ESMTP id T055a0k4FJpKSs for <vger@sl-h.de>;
	Thu, 15 May 2008 21:51:26 +0200 (MEST)
	(envelope-from: <stable-commits-owner@vger.kernel.org>)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1761088AbYEOTu6 (ORCPT <rfc822;vger@sl-h.de>);
	Thu, 15 May 2008 15:50:58 -0400
Received: (majordomo@vger.kernel.org) by vger.kernel.org id S1754486AbYEOTu5
	(ORCPT <rfc822;stable-commits-outgoing>);
	Thu, 15 May 2008 15:50:57 -0400
Received: from mx2.suse.de ([195.135.220.15]:54427 "EHLO mx2.suse.de"
	rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
	id S1760931AbYEOTu4 (ORCPT <rfc822;stable-commits@vger.kernel.org>);
	Thu, 15 May 2008 15:50:56 -0400
Received: from Relay2.suse.de (mail2.suse.de [195.135.221.8])
	(using TLSv1 with cipher DHE-RSA-AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mx2.suse.de (Postfix) with ESMTP id 48EE145E9B;
	Thu, 15 May 2008 21:50:55 +0200 (CEST)
Subject: patch usb-remove-picdem-fs-usb-demo-device-from-ldusb.patch added to 2.6.25-stable tree
To: xiaofanc@gmail.com,
 MHund@LD-Didactic.de,
 gregkh@suse.de
Cc: <stable@kernel.org>,
 <stable-commits@vger.kernel.org>
From:	<gregkh@suse.de>
Date:	Thu, 15 May 2008 12:04:53 -0700
In-Reply-To: <200805141920.m4EJKpaj019728@hera.kernel.org>
Message-Id: <20080515195054.27CEB14540CB@imap.suse.de>
Sender:	stable-commits-owner@vger.kernel.org
Precedence: bulk
Reply-To: linux-kernel@vger.kernel.org
List-ID: <stable-commits.vger.kernel.org>
X-Mailing-List:	stable-commits@vger.kernel.org
Status: R
X-Status: NC
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  


This is a note to let you know that we have just queued up the patch titled

     Subject: USB: remove PICDEM FS USB demo (04d8:000c) device from ldusb Subject: USB: remove PICDEM FS USB demo (04d8:000c) device from ldusb

to the 2.6.25-stable tree.  Its filename is

     usb-remove-picdem-fs-usb-demo-device-from-ldusb.patch

A git repo of this tree can be found at 
    http://www.kernel.org/git/?p=linux/kernel/git/stable/stable-queue.git;a=summary


>From stable-bounces@linux.kernel.org Wed May 14 12:21:11 2008
From: Xiaofan Chen <xiaofanc@gmail.com>
Date: Wed, 14 May 2008 19:20:51 GMT
Subject: USB: remove PICDEM FS USB demo (04d8:000c) device from ldusb
To: jejb@kernel.org, stable@kernel.org
Message-ID: <200805141920.m4EJKpaj019728@hera.kernel.org>

From: Xiaofan Chen <xiaofanc@gmail.com>

commit 5fc89390f74ac42165db477793fb30f6a200e79c upstream
Date: Tue, 13 May 2008 21:52:00 +0800
Subject: USB: remove PICDEM FS USB demo (04d8:000c) device from ldusb

Microchip has changed the PICDEM FS USB demo device (0x04d8:000c)
to use bulk transfer and not interrupt transfer. So I've updated the libusb
based program here (Post #31).
	http://forum.microchip.com/tm.aspx?m=106426&mpage=2

So I believe that the in-kernel ldusb driver will no longer work with the
demo firmware.  It should be removed.


Signed-off-by: Xiaofan Chen <xiaofanc@gmail.com>
Cc: Michael Hund <MHund@LD-Didactic.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/misc/ldusb.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/usb/misc/ldusb.c
+++ b/drivers/usb/misc/ldusb.c
@@ -63,9 +63,6 @@
 #define USB_DEVICE_ID_VERNIER_CYCLOPS	0x0004
 #define USB_DEVICE_ID_VERNIER_LCSPEC	0x0006
 
-#define USB_VENDOR_ID_MICROCHIP		0x04d8
-#define USB_DEVICE_ID_PICDEM		0x000c
-
 #ifdef CONFIG_USB_DYNAMIC_MINORS
 #define USB_LD_MINOR_BASE	0
 #else
@@ -92,7 +89,6 @@ static struct usb_device_id ld_usb_table
 	{ USB_DEVICE(USB_VENDOR_ID_VERNIER, USB_DEVICE_ID_VERNIER_GOTEMP) },
 	{ USB_DEVICE(USB_VENDOR_ID_VERNIER, USB_DEVICE_ID_VERNIER_SKIP) },
 	{ USB_DEVICE(USB_VENDOR_ID_VERNIER, USB_DEVICE_ID_VERNIER_CYCLOPS) },
-	{ USB_DEVICE(USB_VENDOR_ID_MICROCHIP, USB_DEVICE_ID_PICDEM) },
 	{ USB_DEVICE(USB_VENDOR_ID_VERNIER, USB_DEVICE_ID_VERNIER_LCSPEC) },
 	{ }					/* Terminating entry */
 };


Patches currently in stable-queue which might be from xiaofanc@gmail.com are

queue-2.6.25/usb-remove-picdem-fs-usb-demo-device-from-ldusb.patch
--
To unsubscribe from this list: send the line "unsubscribe stable-commits" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

